appveyor.yml

```diff
@@ -41,6 +41,8 @@ environment:
       ARGS: --toolset=msvc-14.1 address-model=64 cxxflags=-clr asynch-exceptions=on
     - APPVEYOR_BUILD_WORKER_IMAGE: Visual Studio 2017
       ARGS: --toolset=msvc-14.1 address-model=32 cxxflags=-clr asynch-exceptions=on
+    - APPVEYOR_BUILD_WORKER_IMAGE: Visual Studio 2017
+      ARGS: --toolset=msvc-14.1 architecture=arm testing.launcher=../../type_traits/test/null_runner.bat
```

Comment:
Wouldn't `testing.execute=off` suffice here?

Reply:
I didn't know that existed - but yes just what we need!